pelican/log.py

```diff
@@ -92,10 +92,15 @@ class LimitFilter(logging.Filter):
     """
 
     _ignore = set()
+    _raised_messages = set()
     _threshold = 5
     _group_count = defaultdict(int)
 
     def filter(self, record):
+        # don't filter log messages when "debug" is enabled
+        logger_level = logging.getLogger().getEffectiveLevel()
+        if logger_level == logging.DEBUG:
+            return True
```

Comment:
You right. We discussed in IRC a "collision" situation when several messages have one template. I proposed lightweight "protection" with enabling all silenced (LOG_FILTER) messages when `--debug` passed to help user digging into a problem when it arised. My thought was threshold-based filtering can hide potential problems as well as template-based filtering.

I'm not sure what is better, so will give back original behaviour for threshold-based filtering.